### ISSUE TITLE
Set version of `doctrine/annotations` dependency

### ISSUE THREAD

**Issue by AntonShevchuk** [zircote/swagger-php#620]
Hi,
Please set version of `doctrine/annotations` to `~1.6`.
Release number 2.0 is upcoming soon

**Comment by bfanger** [comment]
Lots of packages use doctrine/annotations if i set a version this can block installation of other packages. Does 2.0 break swagger-php?

**Comment by AntonShevchuk** [comment]
Yep. You can't use `swagger-php` with new version, you will received next error:
```
Class 'Doctrine\Common\Annotations\AnnotationRegistry' not found
```

**Comment by bfanger** [comment]
Thanks for the heads up, i've made updated swagger-php to be compatible with both versions, let me know it starts breaking again as 2.0 develops.
